src/Components/WebAssembly/WebAssembly/src/Hosting/IWebAssemblyHostEnvironment.cs

```diff
@@ -13,5 +13,31 @@ public interface IWebAssemblyHostEnvironment
         /// Configured to "Production" when not specified by the host.
         /// </summary>
         string Environment { get; }
+
+        /// <summary>
+        /// Returns true if the current environment is "Production".
+        /// </summary>
+        /// <returns>True if the environment name is "Production"</returns>
+        bool IsProduction();
```

Comment:
Sorry, I should have been clearer. Something like this: https://github.com/dotnet/aspnetcore/blob/4e44025a52e4b73aa17e09a8041b0e166e0c5ce0/src/Hosting/Abstractions/src/HostingEnvironmentExtensions.cs#L12.

Reply:
Done.